keras_cv/layers/preprocessing/random_cutout.py

```diff
@@ -173,3 +214,40 @@ def get_config(self):
         }
         base_config = super().get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+    def _cutout_bounding_boxes(self, image, bounding_boxes, transformation):
+        top = tf.cast(transformation["top"], dtype=self.compute_dtype)
+        left = tf.cast(transformation["left"], dtype=self.compute_dtype)
+        x1, y1, x2, y2, rest = tf.split(
+            bounding_boxes, [1, 1, 1, 1, bounding_boxes.shape[-1] - 4], axis=-1
+        )
+        output = tf.concat(
+            [
+                x1 - left,
+                y1 - top,
+                x2 - left,
+                y2 - top,
+                rest,
+            ],
+            axis=-1,
+        )
+        return output
+
+    def _resize_bounding_boxes(self, image, bounding_boxes):
+        image_shape = tf.shape(image)
+        x_scale = tf.cast(self.width / image_shape[0], dtype=self.compute_dtype)
+        y_scale = tf.cast(self.height / image_shape[0], dtype=self.compute_dtype)
+        x1, y1, x2, y2, rest = tf.split(
+            bounding_boxes, [1, 1, 1, 1, bounding_boxes.shape[-1] - 4], axis=-1
+        )
+        output = tf.concat(
+            [
+                x1 * x_scale,
+                y1 * y_scale,
+                x2 * x_scale,
+                y2 * y_scale,
+                rest,
```

Comment:
we recently updated bounding boxes to be a dictionary format instead of a 5D stacked Tensor, can you remove the `rest` axis and operate on the dictionary format?

This will require rebasing?

Reply:
Sure

Reply:
@LukeWood Will there  be  a problem if  I update all the changes required in this PR via forcepush?

Reply:
> @LukeWood Will there be a problem if I update all the changes required in this PR via forcepush?

Should be fine, its a personal branch